ocaml/message-switch/switch/logging.ml

```diff
@@ -71,21 +71,15 @@ let ignore_fmt_lwt fmt = Printf.ksprintf (fun _ -> Lwt.return ()) fmt
 (* General system logging *)
 let logger = create 512
 
-type level = Debug | Info | Warn | Error | Null
-
-let log_level = ref Warn
+type level = Info | Warn | Error
```

Comment:
It looks like the only purpose is to turn these into strings later. I would assume that function like `info` could pass the correct string directly without using a `level` value which is turned into a string. The one thing this has going for it, it prevents arbitrary strings to be used.